src/index.ts

```diff
@@ -18,28 +18,18 @@ import { SSXServer, SSXExpressMiddleware, SSXInfuraProviderNetworks, SSXRPCProvi
 const FileStoreStore = FileStore(Session);
 
 config();
-const PROD = process.env.ENVIRONMENT === 'production';
-const STAGING = process.env.ENVIRONMENT === 'staging';
 
-if (!process.env.SESSION_COOKIE_NAME || !process.env.SECRET) {
+if (!process.env.SSX_SIGNING_KEY) {
```

Comment:
The only secret the server requires is a signing key, for issuing signed cookies